docs/mmds/mmds-user-guide.md

```diff
@@ -17,7 +28,7 @@ interface configuration API can be found in the
 
 ```bash
 MMDS_NET_IF=eth0
-curl --unix-socket /tmp/firecracker.socket -i                 \
+curl --unix-socket $<span class="x x-first x-last">UNIX_SOCKET</span> -i                 \
```

Comment:
What is the use of the span tag? It appears as literal text within the .md, so it has not effect.

Reply:
some other reviewer suggested this to me.

Reply:
reverted.